### PR TITLE
Running a job

### PR DESCRIPTION
**Pull request by liampm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,5 +14,3 @@ COPY ./go-watch.sh /bin/go-watch.sh
 RUN chmod +x /bin/go-watch.sh
 
 CMD ['go-watch.sh']
-
```
Comment (dom111) [inline Dockerfile:17]: Ooops...

File: src/adapter/controller/task.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			w.WriteHeader(404) // Not found for any invalid IDs
+			return
+		}
+
+		err = marshalJSONResponse(w, controller.ReadRepo.AllForProject(projectUuid))
+		if err != nil {
+			panic(err) // Panic whilst in development
+		}
+
+		w.WriteHeader(200)
```
Comment (dom111) [inline src/adapter/controller/task.go]: `2019/05/24 10:27:56 http: superfluous response.WriteHeader call from github.com/liampm/static-code-analysis-tool/adapter/controller.(*TaskController).All.func1 (task.go:31)`

Not sure if we need this then? Is it handled by the caller?

File: src/domain/task.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+import (
+	uuid "github.com/satori/go.uuid"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+type TaskInstance struct {
+	Id        uuid.UUID `json:id`
```
Comment (dom111) [inline src/domain/task.go]: These field names need quoting (I think that'll fix the capitalisation in the JSON responses)

**Review by dom111 — CHANGES_REQUESTED** [review]

File: src/domain/task.go (hunk 3 above)
Comment (liampm) [reply src/domain/task.go]: ah yes! good spot

File: src/adapter/controller/task.go (hunk 2 above)
Comment (liampm) [reply src/adapter/controller/task.go]: It should be moved above the calls to write (done by marshalJSONResponse)
It's harmless in this instance but should move

File: Dockerfile (hunk 1 above)
Comment (liampm) [reply Dockerfile:17]: ah, it's just whitespace

**Pull request merged**